stubs/third_party_taint/tornado_sources_sinks.pysa

```diff
@@ -45,3 +45,22 @@ tornado.httputil.HTTPServerRequest.body: TaintSource[UserControlled, UserControl
 tornado.httputil.HTTPServerRequest.arguments: TaintSource[UserControlled, UserControlled_Payload]
 tornado.httputil.HTTPServerRequest.query_arguments: TaintSource[UserControlled, UserControlled_Payload]
 tornado.httputil.HTTPServerRequest.body_arguments: TaintSource[UserControlled, UserControlled_Payload]
+
+ModelQuery(
+  find = "methods",
+  where = [
+    parent.extends("tornado.web.RequestHandler", is_transitive=True),
+    AnyOf(
+      name.matches("\.get+$"),
```

Comment:
Why do we have the `+` here? That just makes the `t` in `get` optional:
![image](https://user-images.githubusercontent.com/2495654/125828886-e0d984fc-80f0-41cd-9f53-edf9ba1fbed8.png)

We should probably remove it.

```suggestion
      name.matches("\.get$"),
```

Reply:
Tbh, I am not so good in regex, I tried a generator just like the one in the pic 😅, I did try out few combinations but not ge 😅, it seems you are right! I will make the change!